.ci/jobs/e2e-custom.yml

```diff
@@ -0,0 +1,30 @@
+---
+- job:
+    description: Job that runs e2e tests against custom ECK image running in a dedicated k8s cluster in GKE. This Job is managed by JJB.
+    logrotate:
+      daysToKeep: 7
+      numToKeep: 100
+      artifactDaysToKeep: 5
+      artifactNumToKeep: 10
+    name: cloud-on-k8s-e2e-tests-custom
+    project-type: pipeline
+    parameters:
+      - string:
+          name: IMAGE
+          description: "Docker image with ECK"
+      - string:
+          name: VERSION
+          default: 1.11
```

Comment:
Isn't `1.12` our new defaut k8s version? (https://github.com/elastic/cloud-on-k8s/pull/1167)

Reply:
Ahh ok, I can update it.
btw, simply bumping version in `makefile` isn't enough for CI. In some jobs k8s version are set explicitly in Jenkins pipeline(s)

Reply:
Fixed